index.js

```diff
@@ -15,34 +16,29 @@ const isLink = (object) => object && object.sys && object.sys.type === 'Link';
  */
 const findNormalizableLinkInArray = (array, predicate) => {
```

Comment:
the whole function might be replaced by an `Array.find()`? 🤔

Reply:
Which is, on my machine, 92% slower so... probably not an smart option:
https://jsperf.com/array-find-vs-for